proto/proto/store/rpc.proto

```diff
@@ -149,6 +149,8 @@ message AccountProofRequest {
 
     // Block at which we'd like to get this data. If present, must be close to the chain tip.
     // If not present, data from the latest block will be returned.
+    // If the specified block does not contain an update for the specified account,
+    // the latest available data will be returned.
```

Comment:
I think this is a bit ambiguous and could mean that asking for block `N` could return data after `N` aka `latest > N`.

I think phrasing the entire comment as something like

```
// Optional block height at which to return the proof.
//
// Defaults to current chain tip if unspecified.
```